client/packages/common/src/ui/layout/tables/components/Cells/CurrencyCell.tsx

```diff
@@ -27,7 +27,7 @@ export const CurrencyCell = <T extends RecordWithId>({
   CellProps<T>
 > => {
   const { c } = useCurrency(currencyCode);
-  const price = Number(column.accessor({ rowData })) ?? 0;
+  const price = Number(column.accessor({ rowData }) ?? 0);
```

Comment:
Ooooh, I was like "Didn't I just do this yesterday?", but I see it's the CurrencyCell, not the CurrencyInput. 🙄

Thanks 🙏